profiles/pminio/README.md

```diff
@@ -0,0 +1,30 @@
+# pminio
+
+## Usage
+
+Adding this profile sets up a Minio instance that is configured as Pulp's S3 backend. The Minio
+instance is launched as a separate service and can be accessed inside the main container at
+`http://pminio:9000`. The Minio Client CLI,`mc`, is also installed and configured inside the main 
+container to talk to the Minio service.
+
+## Extra Variables
+
+Below are the default variables used in configuring the Minio instance. If you are using
+multiple OCI environments you must change the `MINIO_PORT` and `MINIO_CONSOLE_PORT` in your
+`compose.env` profiles to allow the multiple services to bind correctly.
+
+- `S3_ENDPOINT_URL`
+    - Description: The internal url the Minio instance will be available at.
+    - Default: http://pminio:9000
+- `S3_ACCESS_KEY`
+    - Description: The S3 access key used to access the Minio instance.
+    - Default: pulpminioaccesskey
+- `S3_SECRET_KEY`
+    - Description: The S3 secret key used to access the Minio instance.
+    - Default: pulpminioinsecuresecretkey
+- `MINIO_PORT`
+    - DESCRIPTION: The outside port that the Minio instance will be exposed at.
+    - DEFAULT: 9000
+- `MINIO_CONSOLE_PORT`
```

Comment:
Also, it might be worth adding a note about "DEFAULT_FILE_STORAGE". So, in the end, the settings file (`/etc/pulp/settings.py`) will look like this:
```
CONTENT_ORIGIN='http://localhost:80'
S3_ENDPOINT_URL="http://pminio:9000"
S3_ACCESS_KEY="pulpminioaccesskey"
S3_SECRET_KEY="pulpminioinsecuresecretkey"
PULP_AWS_ACCESS_KEY_ID=S3_ACCESS_KEY
PULP_AWS_SECRET_ACCESS_KEY=S3_SECRET_KEY
PULP_AWS_STORAGE_BUCKET_NAME="pulp"
PULP_AWS_S3_SIGNATURE_VERSION="s3v4"
PULP_AWS_S3_ADDRESSING_STYLE="path"
PULP_AWS_S3_ENDPOINT_URL=S3_ENDPOINT_URL
PULP_DEFAULT_FILE_STORAGE="storages.backends.s3boto3.S3Boto3Storage"

MINIO_PORT="9000"
MINIO_CONSOLE_PORT="9090"
```